config/index.js

```diff
@@ -44,9 +44,7 @@ module.exports = {
 			browserCallbackUri: 'https://www.kiva.org/process-browser-auth',
 			serverCallbackUri: 'https://www.kiva.org/process-ssr-auth',
 			domain: 'login.kiva.org',
-			scope: 'https://www.kiva.org/last_login ' +
-				'https://www.kiva.org/kiva_id ' +
-				'openid email profile',
+			scope: 'openid mfa',
```

Comment:
The `email` and `profile` scopes add PII to the id token, and it isn't being used, so I removed those scopes.
The `last_login` and `kiva_id` scopes aren't needed because those claims are always added to the tokens no matter what scope is requested.